docs/content/users/extend/additional-services.md

```diff
@@ -79,29 +79,29 @@ For example,
-* [Beanstalkd](https://github.com/ddev/ddev-beanstalkd): `ddev get ddev/ddev-beanstalkd`.
-* [Browsersync](https://github.com/ddev/ddev-browsersync): `ddev get ddev/ddev-browsersync`.
-* [cron](https://github.com/ddev/ddev-cron): `ddev get ddev/ddev-cron`.
-* [Elasticsearch](https://github.com/ddev/ddev-elasticsearch): `ddev get ddev/ddev-elasticsearch`.
-* [Memcached](https://github.com/ddev/ddev-memcached): `ddev get ddev/ddev-memcached`.
-* [MongoDB](https://github.com/ddev/ddev-mongo): `ddev get ddev/ddev-mongo`.
-* [PDFreactor](https://github.com/ddev/ddev-pdfreactor): `ddev get ddev/ddev-pdfreactor`
-* [Platform.sh](https://github.com/ddev/ddev-platformsh): `ddev get ddev/ddev-platformsh`
-* [Proxy Support](https://github.com/ddev/ddev-proxy-support): `ddev get ddev/ddev-proxy-support`.
-* [Redis Commander](https://github.com/ddev/ddev-redis-commander): `ddev get ddev/ddev-redis-commander`.
-* [Redis](https://github.com/ddev/ddev-redis): `ddev get ddev/ddev-redis`.
-* [Selenium Standalone Chrome](https://github.com/ddev/ddev-selenium-standalone-chrome): `ddev get ddev/ddev-selenium-standalone-chrome`.
-* [Varnish](https://github.com/ddev/ddev-varnish): `ddev get ddev/ddev-varnish`.
+* [Adminer](https://github.com/ddev/ddev-adminer): `ddev add-on get ddev/ddev-adminer`.
+* [Apache Solr for Drupal](https://github.com/ddev/ddev-drupal-solr): `ddev add-on get ddev/ddev-drupal-solr`.
+* [Beanstalkd](https://github.com/ddev/ddev-beanstalkd): `ddev add-on get ddev/ddev-beanstalkd`.
+* [Browsersync](https://github.com/ddev/ddev-browsersync): `ddev add-on get ddev/ddev-browsersync`.
+* [cron](https://github.com/ddev/ddev-cron): `ddev add-on get ddev/ddev-cron`.
+* [Elasticsearch](https://github.com/ddev/ddev-elasticsearch): `ddev add-on get ddev/ddev-elasticsearch`.
+* [Memcached](https://github.com/ddev/ddev-memcached): `ddev add-on get ddev/ddev-memcached`.
+* [MongoDB](https://github.com/ddev/ddev-mongo): `ddev add-on get ddev/ddev-mongo`.
+* [PDFreactor](https://github.com/ddev/ddev-pdfreactor): `ddev add-on get ddev/ddev-pdfreactor`
+* [Platform.sh](https://github.com/ddev/ddev-platformsh): `ddev add-on get ddev/ddev-platformsh`
+* [Proxy Support](https://github.com/ddev/ddev-proxy-support): `ddev add-on get ddev/ddev-proxy-support`.
+* [Redis Commander](https://github.com/ddev/ddev-redis-commander): `ddev add-on get ddev/ddev-redis-commander`.
+* [Redis](https://github.com/ddev/ddev-redis): `ddev add-on get ddev/ddev-redis`.
+* [Selenium Standalone Chrome](https://github.com/ddev/ddev-selenium-standalone-chrome): `ddev add-on get ddev/ddev-selenium-standalone-chrome`.
+* [Varnish](https://github.com/ddev/ddev-varnish): `ddev add-on get ddev/ddev-varnish`.
 
 ## Managing Installed Add-Ons
 
-Add-ons installed in DDEV v1.22+ are versioned and can be viewed by running `ddev get --installed`.
+Add-ons installed in DDEV v1.22+ are versioned and can be viewed by running `ddev add-on list --installed`.
```

Comment:
```suggestion
Installed add-ons are versioned and can be viewed by running `ddev add-on list --installed`.
```

Reply:
That's not really related to this PR but it's a tiny change so I'll make it.

Reply:
It's important to improve docs as we go on all these things. And of course since I gave the suggestion and you can just click a button to add it it's not too hard to fix.

<img width="799" alt="image" src="https://github.com/user-attachments/assets/aa40df9d-11db-4e6b-a110-77f6582e04e4">

Reply:
> it's not too hard to fix.

That's why I did make this change :p